climakitae/data_export.py

```diff
@@ -34,7 +34,7 @@ def export_to_csv(data_to_export,save_name,**kwargs):
     kwargs: reserved for future use
     '''
     
-    print("WARNING: Exporting to CSV can take a long time,"+
+    print("Exporting to CSV can take a long time,"+
                  " and is not recommended for data with more than 2 dimensions."+
                   " Please note that the data will be compressed via gzip. Even so,"+
          " inherent inefficiencies may result in a file which is too large to save here.")
```

Comment:
I recommend removing this statement. I tested with several different data types and the data downloads were all almost instantaneous